### PR TITLE
[CB] Fix sysadmin delete course

### PR DESCRIPTION
**Pull request by arsentur**
**Description**
Processed 500 error in sysadmin dashboard, when try to delete non existing course by ID or dir. 

**YouTrack** 
https://youtrack.raccoongang.com/issue/CB-82

### CODE REVIEW

**Comment by svic** [comment]
start all tests

**Comment by svic** [comment]
start all tests

**Comment by svic** [comment]
start all tests

**Comment by svic** [comment]
start javascript tests

File: lms/djangoapps/dashboard/sysadmin.py [hunk 1]
```diff
@@ -482,7 +482,15 @@ def post(self, request):
...
         elif action == 'del_course':
             course_id = request.POST.get('course_id', '').strip()
-            course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+
+            try:
+                course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+            except InvalidKeyError as err:
+                course_key = course_id
+                self.msg += _(
+                    'Error - cannot get course with ID {0}<br/><pre>{1}</pre>'
```
Comment (idegtiarov) [inline lms/djangoapps/dashboard/sysadmin.py]: Shouldn't we wrap msg with the tag like <p> or similar?

**Pull request merged**